bot/reddit.py

```diff
@@ -14,7 +14,7 @@
 )
 
 
-_USERNAME_RE = compile(r"/u/[A-Za-z0-9_-]+")
+_USERNAME_RE = compile(r"[ | /]u/[A-Za-z0-9_-]+")
```

Comment:
I fuckin hate regex, can you explaint this please? And add some example matches as comments just above the regex its self 😅

Reply:
Also elaborate on what the effect is - I see we're slicing differently below.